### PR TITLE
Added Dark theme

### PR DESCRIPTION
**Pull request by helloitsm3**

### CODE REVIEW

**Comment by wslyvh** [comment]
This is great! It was recently requested again on Twitter as well.. 

The PR is a bit difficult to review tho, because I guess you have auto formatting turned on? 
I also noticed the sidenav isn't switching to dark? Was that intentional? 
You can see the preview at https://useweb3-kqql0c15o-useweb3.vercel.app/ 

I do love the color scheme tho! :)

**Comment by helloitsm3** [comment]
> This is great! It was recently requested again on Twitter as well.. 
> 
> 
> 
> The PR is a bit difficult to review tho, because I guess you have auto formatting turned on? 
> 
> I also noticed the sidenav isn't switching to dark? Was that intentional? 
> 
> You can see the preview at https://useweb3-kqql0c15o-useweb3.vercel.app/ 
> 
> 
> 
> I do love the color scheme tho! :) 

Yup, I have auto formatting on and yes the I didn't change the sidenav because I couldn't find the right colour scheme for it 😅

**Comment by helloitsm3** [comment]
This is how it looks now.

Light theme:
image

Dark theme:
image

**Comment by helloitsm3** [comment]
Updated readme with some instructions for #16

**Comment by helloitsm3** [comment]
Added more theme options. Now it supports Pantone and Blueberry Dark

Pantone
image

Blueberry Dark
image

**Comment by wslyvh** [comment]
> > This is great! It was recently requested again on Twitter as well..
> > The PR is a bit difficult to review tho, because I guess you have auto formatting turned on?
> > I also noticed the sidenav isn't switching to dark? Was that intentional?
> > You can see the preview at https://useweb3-kqql0c15o-useweb3.vercel.app/
> > I do love the color scheme tho! :)
> 
> Yup, I have auto formatting on and yes I didn't change the sidenav because I couldn't find the right color scheme for it sweat_smile I'll be adding the color for the sidenav later today

Ok, so this isn't great, but the auto formatting made it really difficult to review.
It also applied completely different styling than the rest of the project.
So I just added a prettier config, so it'll be consistent.

That means you'll likely have some conflicts now. Sorry about that! 
Not sure if you want to fix, or maybe start a new PR and apply your changes again? 

I do love the dark mode and appreciate your work!

**Comment by wslyvh** [comment]
And maybe we should also darken the cards? They still have their white backgrounds :)

**Comment by helloitsm3** [comment]
> > > This is great! It was recently requested again on Twitter as well..
> 
> > > The PR is a bit difficult to review tho, because I guess you have auto formatting turned on?
> 
> > > I also noticed the sidenav isn't switching to dark? Was that intentional?
> 
> > > You can see the preview at https://useweb3-kqql0c15o-useweb3.vercel.app/
> 
> > > I do love the color scheme tho! :)
> 
> > 
> 
> > Yup, I have auto formatting on and yes I didn't change the sidenav because I couldn't find the right color scheme for it sweat_smile I'll be adding the color for the sidenav later today
> 
> 
> 
> Ok, so this isn't great, but the auto formatting made it really difficult to review.
> 
> It also applied completely different styling than the rest of the project.
> 
> So I just added a prettier config, so it'll be consistent.
> 
> 
> 
> That means you'll likely have some conflicts now. Sorry about that! 
> 
> Not sure if you want to fix, or maybe start a new PR and apply your changes again? 
> 
> 
> 
> I do love the dark mode and appreciate your work!

Thank you. I'll fix the changes on my end

**Comment by wslyvh** [comment]
Sorry, this PR got so messy. Build fails (on Vercel) because of all these style errors..
Need to take a proper look, hopefully later this week, to see if we can fix this.

File: src/components/sitenav.tsx [hunk 1]
```diff
@@ -63,5 +63,5 @@ export function Sitenav(props: Props) {
         </li>
       </ul>
     </nav>
-  )
+  );
 }
```
Comment (wslyvh) [inline src/components/sitenav.tsx:67]: This file has wrong formatting. Also no updates.
Please undo/revert changes.

File: src/hooks/useNavigation.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  const context = useContext(NavigationContext);
 
   if (context === undefined) {
-    throw new Error('useNavigation must be used within a NavigationProvider')
+    throw new Error("useNavigation must be used within a NavigationProvider");
   }
 
-  return context
+  return context;
 }
```
Comment (wslyvh) [inline src/hooks/useNavigation.tsx:12]: This file has wrong formatting. Also no updates.
Please undo/revert changes.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,75 @@
...
+    </div>
+    <br />
+    <p>
+        useWeb3 is a learning platform for developers to explore and learn about Web3. Whether you’re a new dev getting your hands dirty for the first time, or a seasoned developer making the transition into the Web3 space. 
+        <br />
+        <br />
+        Explore the latest resources, tutorials, challenges, tools, courses and boilerplates and start learning. Once you’re ready, browse the job board to land a job at some of the leading companies that work on core, open-source infrastructure, products, tools, frameworks, DAO's, etc. Explore. Learn. Build.
+    </p>        
+  </div>
+</div>
```
Comment (wslyvh) [inline README.md:19]: Can you please remove the html? Just markdown

File: src/components/dropdown.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import { useRef, useState } from 'react'
-import { useOnOutsideClick } from 'hooks/useOnOutsideClick'
-import styles from './dropdown.module.scss'
+import { useRef, useState } from "react";
+import { useOnOutsideClick } from "hooks/useOnOutsideClick";
+import styles from "./dropdown.module.scss";
 
 interface Props {
   items: Array<string>
```
Comment (wslyvh) [inline src/components/dropdown.tsx:6]: This file has wrong formatting. Also no updates.
Please undo/revert changes.

File: src/components/filtered-overview.tsx [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-import { Featured } from './featured'
+import { useState, useEffect } from "react";
+import { ContentItem } from "types/content-item";
+import styles from "pages/pages.module.scss";
+import { Card } from "./card";
+import { Dropdown } from "./dropdown";
+import { Featured } from "./featured";
 
 interface Props {
   title: string
```
Comment (wslyvh) [inline src/components/filtered-overview.tsx:9]: This file has wrong formatting. Also no updates.
Please undo/revert changes.

File: src/components/card.tsx [hunk 6]
```diff
@@ -40,7 +40,7 @@ export function Card(props: Props) {
         {!props.detailsUrl && <span></span>}
         {props.url && (
           <Link href={props.url}>
-            <button className="accent block">&raquo;</button>
+            <button className='accent block'>&raquo;</button>
           </Link>
         )}
       </div>
```
Comment (wslyvh) [inline src/components/card.tsx:46]: This file has wrong formatting. Also no updates.
Please undo/revert changes.

File: src/pages/_document.tsx [hunk 7]
```diff
@@ -13,6 +13,6 @@ export default class AppDocument extends Document {
           <NextScript />
         </body>
       </Html>
-    )
+    );
   }
 }
```
Comment (wslyvh) [inline src/pages/_document.tsx:18]: This file has wrong formatting. Also no updates.
Please undo/revert changes.

**Review by wslyvh — COMMENTED** [review]
Sorry, this PR got so messy. Build fails (on Vercel) because of all these style errors..

Please try not to commit files that don't really include any changes. 
Some are just updating single quotes to double, adding semicolons, etc. Just style/formatting. 
I wonder if it might be easier to open a new PR, with just the changes you've actually made.

**Comment by helloitsm3** [comment]
I agree, this is getting too messy. I'll make a new PR

**Pull request closed without merging**